lib/bunq/resource.rb

```diff
@@ -101,13 +101,19 @@ def verify_and_handle_response(response, request, result, &block)
     end
 
     def handle_response(response, _request, _result, &block)
+      if Bunq::configuration.sandbox && response.code == 409
```

Comment:
Is there a reason why this is not in the case statement?

Reply:
a conditional inside a case statement only works when it's `case` instead of `case response.code`. Since using `case` would result in code like `when response.code == 200 || response.code == 201` I went with a `if` statement now. I hope that's also fine :)